lib/trino-orc/src/main/java/io/trino/orc/reader/SliceDictionaryColumnReader.java

```diff
@@ -363,6 +363,12 @@ public void close()
     @Override
     public long getRetainedSizeInBytes()
     {
-        return INSTANCE_SIZE;
+        return INSTANCE_SIZE + sizeOf(nonNullValueTemp) + sizeOf(nonNullPositionList) + sizeOf(dictionaryData)
```

Comment:
there are 6 arrays in this class, only 5 are accounted for

Reply:
done, added `currentDictionaryData`.

Reply:
This is still not perfect as there are many more objects in `SliceDictionaryColumnReader`, but I think it's fine for now